src/Microsoft.Sbom.Constants/SpdxConstants.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using Microsoft.Sbom.Contracts;
+using Microsoft.Sbom.Contracts.Enums;
+using Microsoft.Sbom.Extensions.Entities;
+
+namespace Microsoft.Sbom.Constants;
+
+public static class SpdxConstants
+{
+    public const string ManifestFolder = "_manifest";
+    public const string LoggerTemplate = "##[{Level:w}]{Message}{NewLine}{Exception}";
+
+    public static ManifestInfo SPDX22ManifestInfo = new ManifestInfo
+    {
+        Name = "SPDX",
+        Version = "2.2"
+    };
+
+    public static ManifestInfo SPDX30ManifestInfo = new ManifestInfo
+    {
+        Name = "SPDX",
+        Version = "3.0"
+    };
+
+    public static Collection<SbomSpecification> SupportedSbomSpecifications = new()
+    {
+        ManifestInfo.ToSBOMSpecification(SPDX22ManifestInfo),
+        ManifestInfo.ToSBOMSpecification(SPDX30ManifestInfo),
+    };
+
+    public static Collection<ManifestInfo> SupportedSpdxManifests = new()
+    {
+        SPDX22ManifestInfo,
+        SPDX30ManifestInfo,
+    };
+
+    // TODO: move to test csproj
```

Comment:
This TODO seems to be undone

Reply:
This is a TODO made by someone else that was already in the repo, I'm not sure what they meant but can look into it separately